codegen/smithy-ruby-codegen/src/test/java/software/amazon/smithy/ruby/codegen/RubyCodeWriterTest.java

```diff
@@ -0,0 +1,72 @@
+package software.amazon.smithy.ruby.codegen;
```

Comment:
Our pattern hasn't been to write java tests. We've instead written our tests in ruby (integration-specs) against the sample model, and done expectations there. I appreciate the bias for action on this one.. however.. could we instead port this as a ruby test so we have one location where we're testing all this stuff?

Reply:
Not sure whether the ruby tests would be able to test this. The java unit test tests the behaviour of the add/close module methods but the generated ruby code remains the same.